src/components/SumField.js

```diff
@@ -11,44 +11,40 @@ class SumFields extends React.Component {
       time: 0,
     }
     setInterval(() => this.incrementTimer(), 1000)
-  }  
-  componentDidMount() {
-        this.computeSalarySum();
-    }
-    render () {
-        return (
-            <div>
-                <h1>Meeting cost: { this.costPerSecond() }</h1>
-                <h3>Elapsed time: { this.renderElapsedTime() } seconds</h3>
-                <h3>Cost per hour: {this.computeSalarySum()}</h3>
-                
-            </div>
-        )
-    }
-    computeSalarySum = () => {
-      const salarySum = this.props.salaries.reduce((sum, salary) => sum + salary);
-      this.props.setSalarySum(salarySum);
-        return(
-            this.props.salarySum
-        );
-    };
-    incrementTimer(){
-      if(!this.props.timerShouldRun) return
-      const newTime = this.state.time+1
-      this.setState({
-        time: newTime
-      })
-      this.props.incrementTimer();
-    }
-    renderElapsedTime() {
+  }
+  render () {
       return (
-        this.props.elapsedTime
+          <div>
+              <h1>Meeting cost: { this.costPerSecond() }</h1>
+              <h3>Elapsed time: { this.renderElapsedTime() } seconds</h3>
+              <h3>Cost per hour: {this.computeSalarySum()}</h3>
+          </div>
       )
-    }
-    costPerSecond = () => {
-      const costPerSecond = this.props.salarySum / 60 / 60;
-      return ((costPerSecond * this.props.elapsedTime).toFixed(2))
-    }
+  }
+  computeSalarySum = () => {
+    const salarySum = this.props.salaries.reduce((sum, salary) => sum + salary);
+    this.props.updateSalarySum(salarySum);
+      return(
+          this.props.salarySum
+      );
+  };
+  incrementTimer(){
+    if(!this.props.timerShouldRun) return
+    const newTime = this.state.time+1
+    this.setState({
+      time: newTime
+    })
+    this.props.incrementTimer();
+  }
+  renderElapsedTime() {
+    return (
+      this.props.elapsedTime
+    )
+  }
+  costPerSecond = () => {
```

Comment:
This is very nitpicky, but I'd just like to suggest a way I think 'cleans up' this function and makes it more readable without knowing the entire context of the program :)
```
calculateCostPerSecond = () => { // add calculate to clarify the intent of the function
  const costPerSecond = this.props.hourlySalarySum / SECONDS_PER_HOUR; // no 'magic numbers'
  const durationInSeconds = this.props.elapsedTime; // ideally I'd change this.props.elapsedTime => durationInSeconds to clarify the intent of the variable
  const totalMeetingCost = costPerSecond * durationInSeconds; //easier to understand the next line when extracting calculations to own variables
  return this.roundToTwoDigits(totalMeetingCost) //easier to read an internal function name than to understand what .toFixed() does (though one could argue developers should know the .js api...)
}

roundToTwoDigits = (numberToRound: number) => {
  return numberToRound.toFIxed(2);
}
```